### ISSUE TITLE
Pylint has no JSON output when no errors are found, which messes up json.loads

### ISSUE THREAD

**Issue by Martmists-GH** [ContinuumIO/ciocheck#47]
https://github.com/ContinuumIO/ciocheck/blob/master/ciocheck/linters.py#L46

An easy fix would be to do this:

```python
    def _parse_json(self, string):
        """Parse output with json keys."""
        if not string:
            return []  # No response, successful
        data = json.loads(string)
        ...
```
